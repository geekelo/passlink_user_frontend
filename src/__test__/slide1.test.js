import React from 'react';
import { render } from '@testing-library/react';
import SlideOne from '../homePage/javascripts/slide1';

test('renders SlideOne correctly', () => {
  const { asFragment } = render(<SlideOne />);
  
  // Initial render snapshot
  expect(asFragment()).toMatchSnapshot();
});
