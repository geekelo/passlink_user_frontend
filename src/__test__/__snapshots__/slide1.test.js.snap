// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SlideOne correctly 1`] = `
<DocumentFragment>
  <div
    class="body-slide"
  >
    <div
      class="slide-content"
    >
      <h1
        class="title"
      >
        Find online & onsite events on PassLink
      </h1>
      <div
        class="ticks-cta-section"
      >
        <div
          class="ticks-section"
        >
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Enhance your knowledge, skills, and career prospects
            </span>
          </p>
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Participate in celebrations & social gatherings
            </span>
          </p>
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Engage extensive collection of religious events
            </span>
          </p>
          <p>
            <span
              class="ticks"
            >
              ✔
            </span>
            <span>
              Access mentorship sessions & programs
            </span>
          </p>
        </div>
        <button
          class="btn"
          type="submit"
        >
          <a
            href="v.com"
          >
            <span>
              FIND EVENTS  
            </span>
            <i
              class="material-icons"
            >
              travel_explore
            </i>
          </a>
        </button>
      </div>
    </div>
    <div
      class="infographic-section"
    >
      <img
        alt="infographic"
        class="infographic"
        src="infographic1.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
